ts/Dashboard/Actions/Bindings.ts

```diff
@@ -101,16 +115,49 @@ class Bindings {
             fireEvent(component, 'mount');
         }
 
-            if (cell && component) {
-                component.setCell(cell);
-                cell.mountedComponent = component;
+        if (cell && component) {
```

Comment:
Shouldn't we also add a `selected` state? IMO after clicking on some KPI, the selection color should be changed.

Reply:
Its active state is added in the click event (a few lines) above.